aten/src/ATen/native/native_functions.yaml

```diff
@@ -1461,6 +1461,9 @@
   dispatch:
     CPU: _neg_out_cpu
     CUDA: _neg_out_cuda
+- func: vectorized_contig_tensor_repeat(Tensor self, int[] repeats) -> Tensor
+  matches_jit_signature: True
```

Comment:
This is true by default and not necessary to add anymore, if the signature is compliant.